### PR TITLE
A0-1115 : Basic test for correct reward points calculation

### PR DESCRIPTION
**Pull request by krzysztofziobro**

### CODE REVIEW

File: e2e-tests/src/test/rewards.rs [hunk 1]
```diff
@@ -107,6 +107,61 @@ fn validators_bond_extra_stakes(config: &Config, additional_stakes: Vec<Balance>
...
+        Some(reserved_members.clone()),
+        Some(non_reserved_members.clone()),
+        Some(4),
+        XtStatus::Finalized,
+    );
+
+    let sessions_per_era = get_sessions_per_era(&connection);
+    let era = wait_for_next_era(&root_connection)?;
+    let start_era_session = era * sessions_per_era;
+    let end_era_session = sessions_per_era * wait_for_next_era(&root_connection)?;
```
Comment (maciejzelaszczyk) [inline e2e-tests/src/test/rewards.rs:132]: Shouldn't this be `start_new_era_session`? Or something like that.

File: e2e-tests/src/test/rewards.rs [hunk 2]
```diff
@@ -107,6 +107,61 @@ fn validators_bond_extra_stakes(config: &Config, additional_stakes: Vec<Balance>
...
+
+    info!(
+        "Checking rewards for sessions {}..{}.",
+        start_era_session, end_era_session
+    );
+
+    for session in start_era_session..end_era_session {
+        let non_reserved_for_session = get_non_reserved_members_for_session(config, session);
+        let members_bench = non_reserved_members
```
Comment (maciejzelaszczyk) [inline e2e-tests/src/test/rewards.rs:141]: There's a function for that: `get_bench_members`.

**Review by maciejzelaszczyk — CHANGES_REQUESTED** [review]
Great work :) Two minor points, other than that LGTM

**Review by piotrMocz — COMMENTED** [review]
I have nothing to add besides what @maciejzelaszczyk commented, in general the PR looks good so I'll accept once you address those comments ;)

**Review by maciejzelaszczyk — APPROVED** [review]
Very nice :)

**Review by piotrMocz — APPROVED** [review]
Cool!

**Pull request merged**